client/src/components/LoginPage.js

```diff
@@ -75,6 +76,26 @@ function LoginPage() {
             </div>
             {signupError && <p className="error-message">Passwords do not match</p>}
```

Comment:
Add toast notification here.

Reply:
Updated hall booking form , added event name option and added some css